homeassistant/components/lock/xiaomi_aqara.py

```diff
@@ -1,4 +1,9 @@
-"""Support for Xiaomi Gateway Lock."""
+"""
+Support for Xiaomi Gateway Lock.
```

Comment:
Could you call it Xiaomi Aqara Lock or similar. I don't like the "Gateway" here.